server/api.js

```diff
@@ -1,9 +1,87 @@
-import { Router } from "express";
+import { response, Router } from "express";
+import query from "./db";
 
 const router = Router();
+const client = query; //just looks a little nicer than query.query
```

Comment:
You can do `import query as client from './db'` instead. Looks a bit cleaner and makes sure imports are at the top